web-app/src/hoc/Layout/container.js

```diff
@@ -57,7 +58,18 @@ export default Wrapped =>
       };
     }
 
+    toggleDrawer = () => {
+      this.setState({ drawerOpen: !this.state.drawerOpen });
+      localStorage.setItem('navDrawerOpen', !this.state.drawerOpen);
```

Comment:
I think
```
this.setState({drawerOpen: !this.state.drawerOpen}, () => localStorage.setItem('navDrawerOpen', this.state.drawerOpen))
```
is safer here. Not sure we can guarantee that the state will (won't?) have been updated when setting the localStorage token.